active/community.md

```diff
@@ -0,0 +1,86 @@
+# Online Community Working Group
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The [Django Forum](https://forum.djangoproject.com/) and the [Django Discord](https://discord.gg/xcRH6mN4fa) are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others.
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+  - ensure consistency that our online Django communities are welcoming and safe spaces
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Moving platforms if required
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- Admin privileges to members of the forum & Discord or existing admin representive joining as a member.
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - Andrew Miller
```

Comment:
I've been a bit busy of late, but now that I have caught up to a bunch of stuff...

> Re Board Liaison – we’ll have four new board members starting soon, who might be interested in this.

I'm going to quote @knyghty who is hopefully still interested here:

> I'm not sure I will have enough time to dedicate to joining the group, but on the off chance I end up on the board I'd be happy to be the board liaison for my term.

Tom, I hope you are still interested :)
 
> Re other members – please let folks know once you think you’re ready to start adding? We’ve sent a few people towards this proposal as part of the DSF board elections.

As a fellow Discord moderator and DSF member, I would be more than happy to join this WG, if you'll have me.

> We have struggled a lot with other groups in selecting a Chair and Co-Chair

+1 to Andy as chair. If possible, a co-chair from another community would _probably_ be good (currently, I guess that means the forums; I am curious whether Reddit should also be involved).

Reply:
In terms of Reddit, that is definitely a possibility, however I would want to have a conversation with the Social Media WG as per this [post](https://www.reddit.com/r/django/comments/1ffew1d/welcome_to_our_new_moderators/) as to whether Reddit is a social media site or a community platform. Essentially I would like to see a clear ownership of different online spaces so that communication doesn't get confused both internally and externally.